### ISSUE TITLE
About loss convergence

### ISSUE THREAD

**Issue by hanq0212** [w86763777/pytorch-ddpm#1]
I tried to train the model with your train code and 2 gpu.
But the training loss seems to converge too fast around 0.04. Is it right?
or Did you use different hyperparameter with the original paper?

Thank you

**Comment by w86763777** [comment]
It seems correct. This is my loss curve.
![loss](https://user-images.githubusercontent.com/9026992/150464219-98e6691e-52da-49ec-8db5-17ebf0aadb57.png)

All the hyperparameter are copied from official implementation.

**Comment by hanq0212** [comment]
Thank you for your quick answer!

**Comment by hanq0212** [comment]
Can I see your FID, IS score logs while training? 
(If you have the record, I want to compare with mine)

Thank you!

**Comment by w86763777** [comment]
I didn't record the FID and IS during training since it costs too much time to generate images for evaluation. However I can give the example at step 10k, 50k and 100k for your reference.
- 10k
![10000](https://user-images.githubusercontent.com/9026992/150496723-3b74cf9b-e9f0-4d80-ae50-4af2d2914148.png)
- 50k
![50000](https://user-images.githubusercontent.com/9026992/150496893-8e4ef672-2a26-4e3e-8c68-3e3e37a6f657.png)
- 100k
![100000](https://user-images.githubusercontent.com/9026992/150496912-80c83a19-6a23-491e-bfd8-125e69a106b5.png)
